RequestJSONBox.module.scss

```diff
@@ -41,7 +40,7 @@
     min-height: 200px;
     padding: 10px 12px;
     border: none;
-    color: var(--copy-color);
+    color: #5d5381;
```

Comment:
Use color variable

Reply:
the color used one time so is it necessary to create a variable for it?